### DISCUSSION TITLE
Any way to install with source rather than dist? (to debug thrown error)

### DISCUSSION THREAD

**Discussion by mitom** [hgouveia/node-downloader-helper#114] [Q&A] [answered]
I'm trying to work with the package in nodejs, and any time an Error is thrown it's incredibly difficult to try to understand why because the installed package is minimised to 1 line, so stacktrace is terrible.

Is there any way to install it with the actual source rather than the minimised version?

Thanks!

**Comment by hgouveia** [comment, marked as answer]
Hello @mitom ,  

you have three options,

1. go to `node_modules/node-downloader-helper/dist/index.js` and just try to beautify it with any prettier

2. if you project is already set up as `type: module`, meaning that you can use `import`,  you could just copy and paste  https://github.com/hgouveia/node-downloader-helper/raw/master/src/index.js this content inside `node_modules/node-downloader-helper/dist/index.js` to replace it

3. by cloning the project and running `npx babel -d ./dist ./src` , and again copying the `index.js` to the `node_modules`, 

just remember to reinstall the library to revert your changes before deploying your app

**Reply by mitom** [reply]
Thank you - I went with option 2. I'm not sure why but when using typescript it meant I had to replace
```
import {DownloaderHelper} from 'node-downloader-helper'
const dl = new DownaloderHelper(...)
```
with
```
import nodeDownloader from 'node-downloader-helper'
const dl = new nodeDownloader.DownaloderHelper(...)
```

i.e. the named export stopped working for some reason. Not that big of a change just FYI if someone else needs it.
